packages/forms/src/styled/file-list/StyledClose.ts

```diff
@@ -0,0 +1,65 @@
+/**
+ * Copyright Zendesk, Inc.
+ *
+ * Use of this source code is governed under the Apache License, Version 2.0
+ * found at http://www.apache.org/licenses/LICENSE-2.0.
+ */
+
+import styled from 'styled-components';
+import {
+  retrieveComponentStyles,
+  getColor,
+  getLineHeight,
+  DEFAULT_THEME
+} from '@zendeskgarden/react-theming';
+
+const COMPONENT_ID = 'forms.file.close';
+
+/**
+ * 1. Reset for <button> element.
+ * 2. Remove dotted outline from Firefox on focus.
+ */
+export const StyledClose = styled.button.attrs({
+  'data-garden-id': COMPONENT_ID,
+  'data-garden-version': PACKAGE_VERSION
+})<any>`
+  display: block;
+  transition: background-color 0.1s ease-in-out, color 0.25s ease-in-out;
+  z-index: 1;
```

Comment:
There's no `position` applied here so I don't think the `z-index` is necessary

Reply:
You're right — after examining the DOM, this element is not a positioned element. Which is why I'm confused at why the `z-index` affects the element 🤔 .

With no `z-index`
<img width="309" alt="Screen Shot 2021-07-06 at 11 20 56 AM" src="https://user-images.githubusercontent.com/1811365/124649566-86c77a00-de4d-11eb-9aee-55d7aae7f43e.png">

With `z-index: 1;`
<img width="410" alt="Screen Shot 2021-07-06 at 11 21 01 AM" src="https://user-images.githubusercontent.com/1811365/124649575-87601080-de4d-11eb-8433-8ae6d4246347.png">

Reply:
Please see https://developer.mozilla.org/en-US/docs/Web/CSS/CSS_Positioning/Understanding_z_index/The_stacking_context#the_stacking_context for the scenarios in which a stacking context is formed.